src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1047,6 +1047,16 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
     {
         _terminal->UserScrollViewport(viewTop);
     }
+    // Method Description:
+    // - Scrolls the viewport of the terminal and updates the scroll bar accordingly
+    // Arguments:
+    // - viewTop: the viewTop to scroll to
+    void TermControl::KeyBoardScrollViewPort(int viewTop)

```

Comment:
_the smallest nit_: 
```suggestion
    void TermControl::KeyBoardScrollViewport(int viewTop)
```

and everywhere else - usually we use "Viewport" as one word